content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -215,7 +215,40 @@ Nevertheless, you must update the list of admission controllers.
 
 ### Azure
 
-You must use `ACS-Engine` to deploy your cluster.
+You can deploy a Kubernetes cluster to Azure via `ACS-Engine` or `AKS` which fully supports Istio as long as Kubernetes 1.9.6 or later is specified.
+
+AKS is a managed Kubernetes service where the master nodes are paid for and managed by Microsoft, no matter the size or number of agent nodes created.  
+
+ACS-Engine provides extensive customization of the Kubernetes cluster being provisioned.  The downloadable `acs-engine` binary processes a simple api-model JSON file into a large Azure resource manager template which automatically deploys a complete Kubernetes cluster.  Master and agent nodes are mangaged entirely and paid for by the customer.
+
+Instructions for AKS:
+
+You can create an AKS cluster via [the `az` cli](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) or [the Azure portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal).  It is only important Kubernetes 1.9.6 or later is specified.  Not all Kubernetes versions are available in all Azure regions.
+
+For the `az` cli option, complete `az login` authentication OR use cloud shell, then run the following commands below.
+
+1. Determine the `Name` for the desired datacenter location
+
+    {{< text bash >}}
+    $ az account list-locations -o table
```

Comment:
This command will return all data centers - even those that currently do not have AKS deployed yet. Consider using the following command which will only bring back those data centers that have AKS deployed.

```
$ az provider list --query "[?namespace=='Microsoft.ContainerService'].resourceTypes[] | [?resourceType=='managedClusters'].locations[]" -o tsv
```

This returns full names (with spaces) so you'll have to add quotes " to location parameters in your subsequent commands.